src/Catalog/packages.config

```diff
@@ -7,10 +7,54 @@
   <package id="Microsoft.Data.Edm" version="5.7.0" targetFramework="net45" />
   <package id="Microsoft.Data.OData" version="5.7.0" targetFramework="net45" />
   <package id="Microsoft.Data.Services.Client" version="5.7.0" targetFramework="net45" />
+  <package id="Microsoft.Extensions.DependencyInjection.Abstractions" version="1.0.0" targetFramework="net45" />
+  <package id="Microsoft.Extensions.Logging" version="1.0.0" targetFramework="net45" />
+  <package id="Microsoft.Extensions.Logging.Abstractions" version="1.0.0" targetFramework="net45" />
+  <package id="Microsoft.NETCore.Platforms" version="1.0.1" targetFramework="net45" />
   <package id="Microsoft.WindowsAzure.ConfigurationManager" version="3.1.0" targetFramework="net45" />
+  <package id="NETStandard.Library" version="1.6.0" targetFramework="net45" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net45" />
-  <package id="NuGet.Versioning" version="3.5.0-rc1-final" targetFramework="net45" />
+  <package id="NuGet.Common" version="4.0.0" targetFramework="net45" />
+  <package id="NuGet.Configuration" version="4.0.0" targetFramework="net45" />
+  <package id="NuGet.Frameworks" version="4.0.0" targetFramework="net45" />
+  <package id="NuGet.Packaging" version="4.0.0" targetFramework="net45" />
+  <package id="NuGet.Packaging.Core" version="4.0.0" targetFramework="net45" />
+  <package id="NuGet.Packaging.Core.Types" version="4.0.0" targetFramework="net45" />
+  <package id="NuGet.Protocol.Core.Types" version="4.0.0" targetFramework="net45" />
+  <package id="NuGet.Protocol.Core.v3" version="4.0.0" targetFramework="net45" />
+  <package id="NuGet.Versioning" version="4.0.0" targetFramework="net45" />
+  <package id="System.Collections" version="4.0.11" targetFramework="net45" />
+  <package id="System.Collections.Concurrent" version="4.0.12" targetFramework="net45" />
+  <package id="System.ComponentModel" version="4.0.1" targetFramework="net45" />
+  <package id="System.Diagnostics.Debug" version="4.0.11" targetFramework="net45" />
+  <package id="System.Diagnostics.Tools" version="4.0.1" targetFramework="net45" />
+  <package id="System.Diagnostics.Tracing" version="4.1.0" targetFramework="net45" />
+  <package id="System.Globalization" version="4.0.11" targetFramework="net45" />
+  <package id="System.IO" version="4.1.0" targetFramework="net45" />
+  <package id="System.IO.Compression" version="4.1.0" targetFramework="net45" />
```

Comment:
https://www.nuget.org/packages/NuGet.Versioning/

When targeting .NET 4.5, there should be no dependencies.